src/main/java/ru/jma/userservice/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package ru.jma.userservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ru.jma.userservice.model.User;
+import ru.jma.userservice.service.UserService;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class UserController {
+
+    @Autowired
+    UserService userService;
+
+    @GetMapping("/user")
+    public List<User> getAllUsers() {
+        return userService.findAll();
```

Comment:
in RestControllers need to return new ResponseEntity<>(userservice.findAll(), HttpStatus.OK);

Reply:
As far as I know ResponseEntity is used only in specific cases. Frequent use of it will clutter up your code